org.eclipse.lemminx/src/main/resources/org/eclipse/lemminx/services/snippets/new-xml-snippets.json

```diff
@@ -0,0 +1,97 @@
+{
+  "New XML with SYSTEM DOCTYPE": {
+    "prefix": [
+      "<!DOCTYPE"
+    ],
+    "suffix": ">",
+    "body": [
+      "<!DOCTYPE ${1:root-element} SYSTEM \"${2:file}.dtd\" >",

```

Comment:
Remove extra space before the closing `>`

Reply:
fixed